### ISSUE TITLE
Opencensus not working with MLFlow

### ISSUE THREAD

**Issue by NastasiaSaby** [census-instrumentation/opencensus-python#1023]
**Describe your environment.** 
- Databricks platform 7.3 LTS
- Python 3.7
- opencensus-ext-azure 1.0.7
- opencensus-ext-logging 0.1.0
- mlflow 1.15.0

**Steps to reproduce.**

```python

import logging

from opencensus.ext.azure.log_exporter import AzureLogHandler

logger = logging.getLogger(__name__)
logger.addHandler(AzureLogHandler(connection_string='InstrumentationKey=d6f4861b-3bf1-4cdb-8ba0-701f940ff21f'))
logger.warning('Hello, World!')

import mlflow

```

**What is the expected behavior?**
 MLFlow should be imported.

**What is the actual behavior?**
It's impossible to import mlflow. Its runs forever.

**Comment by lzchen** [comment]
What is the error message you are getting? How do you know mlflow is not getting imported?

**Comment by NastasiaSaby** [comment]
The scripts never ends. I have no error message.

**Comment by lzchen** [comment]
@NastasiaSaby 
If you remove either `logger.addHandler` or `import mlflow`, the script works?

**Comment by NastasiaSaby** [comment]
@Izchen
Thank you for your answers.
Yes, it works if I remove either logger.addHandler or import mlflow.

**Comment by NastasiaSaby** [comment]
I found a workaround, not the cleanest one though.

I import mlflow at the beginning even if it's not useful this way:

import mlflow
import logging

from opencensus.ext.azure.log_exporter import AzureLogHandler

logger = logging.getLogger(__name__)
logger.addHandler(AzureLogHandler(connection_string='InstrumentationKey=<your-key>'))
logger.warning('Hello, World!')

import mlflow

**Comment by lzchen** [comment]
@NastasiaSaby 
You have two import lines now?

**Comment by NastasiaSaby** [comment]
Yes. In reality, the two import lines in my project are in two different files. But you can reproduce the problem by putting all in the same file. @Izchen
